pkg/store/proxy.go

```diff
@@ -383,78 +393,79 @@ func startStreamSeriesSet(
 				emptyStreamResponses.Inc()
```

Comment:
I don't we want to increment this if the context was actually cancelled.. right?

Reply:
Yes, therefore `numResponses++` only on recv processed.